plugins/org.jboss.tools.docker.reddeer/src/org/jboss/tools/docker/reddeer/core/ui/wizards/RunADockerImagePageOneWizard.java

```diff
@@ -109,17 +101,17 @@ public void setGiveExtendedPrivileges(){
 		setGiveExtendedPrivileges(true);
 	}
 	
-	public void addLinkToOtherContainer(String containerPort, String hostAddress, String hostPort){
+	public void addExposedPort(String containerPort, String hostAddress, String hostPort){
 		new PushButton("Add..").click();
 		new LabeledText("Container port:").setText(containerPort);
 		new LabeledText("Host address:").setText(hostAddress);
 		new LabeledText("Host port:").setText(hostPort);
 		new OkButton().click();
 	}
 	
-	public void addContainerPort(String containerName, String alias){
-		new PushButton("Add..").click();
-		new DefaultCombo("Container").setText(containerName);
+	public void addLinkToContainer(String containerName, String alias){
+		new PushButton(4).click();
```

Comment:
Using id is usually less stable and less descriptive, than using label or some other method. But if you're unable to get that button by label, it's ok.